plugin/github.com.openbao.go.kms.wrapping.plugin.v2_grpc.pb.go

```diff
@@ -18,8 +18,8 @@ import (
 
 // This is a compile-time assertion to ensure that this generated file
 // is compatible with the grpc package it is being compiled against.
-// Requires gRPC-Go v1.32.0 or later.
-const _ = grpc.SupportPackageIsVersion7
+// Requires gRPC-Go v1.64.0 or later.
+const _ = grpc.SupportPackageIsVersion9
```

Comment:
I think this needs a version bump of our grpc dependency. A test is failing saying `SupportPackageIsVersion9` is not defined

Reply:
This should be fine, we already use a later version for the main binary.

Reply:
@JanMa What do you think of this? I've synced all the dependencies from the core repo into this module.

Reply:
@JanMa Ah, never mind, now Transit is failing. lol

Reply:
@JanMa Ok, this should be good. I bumped `api` and `sdk` to match the future `v2.2.0` version.